### PR TITLE
Added docs for case insensitive feature in dq

### PR DESCRIPTION
**Pull request by lsheikal**

### CODE REVIEW

File: docs/userguide/utils/index.rst [hunk 1]
```diff
@@ -89,6 +89,14 @@ variable to ``True`` within the contains api arguments, as shown in examples bel
   >>> output.q.contains('[1,2]', regex=True)
   >>> output.q.contains('.*ware', regex=True)
 
+To do a case insensitive comparison in contains you have to set regex and ignore_case variables to ``True``.
```
Comment (tahigash) [inline docs/userguide/utils/index.rst:92]:
```suggestion
To do a case insensitive comparison in contains you have to set both regex and ignore_case variables to ``True``.
```

File: docs/userguide/utils/index.rst [hunk 2]
```diff
@@ -197,6 +205,17 @@ to True. Examples below elaborate this functionality
   # If searching for both key and value using regex
   >>> output.q.contains_key_value('slot/world_wide_name|mac.*|model', '[a-zA-Z0-9\-\s]+', key_regex=True, value_regex=True)
 
+Similar to contains here also you can do case insensitive comparison.
```
Comment (tahigash) [inline docs/userguide/utils/index.rst:208]:
```suggestion
Similar to ``contains`` here also you can do case insensitive comparison.
```

File: docs/userguide/utils/index.rst [hunk 3]
```diff
@@ -197,6 +205,17 @@ to True. Examples below elaborate this functionality
...
+- If the key have to be case insensitive then you have to set ``key_regex`` and ``ignore_case_key`` as True.
```
Comment (tahigash) [inline docs/userguide/utils/index.rst:209]:
```suggestion
- If the ``key`` has to be case insensitive then you have to set both ``key_regex`` and ``ignore_case_key`` as True.
```

File: docs/userguide/utils/index.rst [hunk 4]
```diff
@@ -197,6 +205,17 @@ to True. Examples below elaborate this functionality
...
+- If the value have to be case insensitive then you have to set ``value_regex`` and ``ignore_case_value`` as True.
```
Comment (tahigash) [inline docs/userguide/utils/index.rst:210]:
```suggestion
- If the ``value`` has to be case insensitive then you have to set both ``value_regex`` and ``ignore_case_value`` as True.
```

File: docs/userguide/utils/index.rst [hunk 5]
```diff
@@ -218,6 +237,8 @@ api.
   # if applying regex only for value set value_regex=True
   >>> output.q.not_contains_key_value('lc', '(3|4)', value_regex=True)
 
+This one also supports case insensitive which was already explained for contains_key_value api.
```
Comment (tahigash) [inline docs/userguide/utils/index.rst:240]:
```suggestion
This one also supports both ``ignore_case_key`` and ``ignore_case_value`` which was already explained for contains_key_value api.
```

**Review by tahigash — APPROVED** [review]
approved with comments

**Review by omehrabi — APPROVED** [review]

File: docs/userguide/utils/index.rst (hunk 1 above)
Comment (lsheikal) [reply docs/userguide/utils/index.rst]: updated

File: docs/userguide/utils/index.rst (hunk 2 above)
Comment (lsheikal) [reply docs/userguide/utils/index.rst]: updated

File: docs/userguide/utils/index.rst (hunk 3 above)
Comment (lsheikal) [reply docs/userguide/utils/index.rst]: updated

File: docs/userguide/utils/index.rst (hunk 4 above)
Comment (lsheikal) [reply docs/userguide/utils/index.rst]: updated

File: docs/userguide/utils/index.rst (hunk 5 above)
Comment (lsheikal) [reply docs/userguide/utils/index.rst]: updated

**Pull request merged**